README.md

```diff
@@ -1,207 +1,339 @@
+| --validator -suggested-fee-recipient | address that receives block fees (0x12345..abcd)                                                                            |
+| --validator -output-dir              | directory where logs are created                                                                                            |
+| --validator -std-output              | set output to console                                                                                                       |
+| --log -folder "./myCustomLogFolder"  | user can setup a custom log directory when starting LUKSO client                                                            |
+| --log -size n                        | Log files capped to the size in MB                                                                                          |
+
+#### Examples:
+
+```bash
+# log files are by default un-capped, be aware that # these files can grow very large.
+# You can use to cap the file size in n MB. Example: # lukso start --log-size 3     to cap the file to 3MB of data
+$ lukso start --log-size n
+# user can set up a custom log directory when starting lukso client
+$ lukso start --log-folder "./myCustomLogFolder"
+# in this case, to access their logs user needs to indicate the folder
+```
+
+## status
+
+Displays the most recent status of LUKSO's node
+
+```bash
+$ lukso status
+```
+
+## stop
+
+Stops all client's activities. usually used when upgrading the client or running maintenance tasks.
+
+```bash
+$ lukso stop
+```
+
+## update
+
+Updates LUKSO client to the latest available version
+
+```bash
+$ lukso update
+```
+
+| Flag             | Description                                                                                                                 |
+| ---------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| --mainnet        | updates LUKSO's mainnet                                                                                                     |
+| --testnet        | updates LUKSO's testnet                                                                                                     |
+| --devnet         | updates LUKSO's devnet                                                                                                      |
+| --geth -\*       | \* Pass any flag to the Geth node [See docs for details](https://geth.ethereum.org/docs/fundamentals/command-line-options)  |
+| --erigon -\*     | \* Pass any flag to the Erigon node [See docs for details](https://github.com/ledgerwatch/erigon)                           |
+| --prysm -\*      | \* Pass any flag to the Prysm node [See docs for details](https://docs.prylabs.network/docs/prysm-usage/parameters)         |
+| --lighthouse -\* | \* Pass any flag to the Lighthouse node [See docs for details](https://lighthouse-book.sigmaprime.io/advanced-datadir.html) |
```

Comment:
Please refer to https://app.clickup.com/t/85zrt1xn1 for a list of relevant flags - we don't need any client-specific ones